man/git-vendor.md

```diff
@@ -11,13 +11,15 @@ git-vendor(1) -- manage vendored dependency subtrees
 
 `git-vendor update <name> [<ref>]`
 
+`git-vendor push <name> [<ref>]`
```

Comment:
add new optional argument that can override `$repository` as in most cases you will not have write access to the origin repo and if you want to upstream your changes you would need to:
1. fork
2. upstream to the fork-repo
3. create pull- request from the fork-repo to the original repo

Reply:
for this I will need to parse the arguments and add the new optional argument with an option name to distinguish between ref and repo like:

git vendor upstream <name>  [**--repo** repository] and optional ref